.ebextensions/nginx-upload-size.config

```diff
@@ -2,3 +2,8 @@ files:
   /etc/nginx/conf.d/proxy.conf:
     content: |
       client_max_body_size 50M;
+
+
+container_commands:
+  01_reload_nginx:
+    command: "service nginx reload"
```

Comment:
Can you please clarify when this command will run? On each deployment or setting up new server? I don't think we need it here, as I understand you only need to run it once after you changed config?